featuretools/tests/entityset_tests/test_es.py

```diff
@@ -2461,3 +2461,47 @@ def test_setitem(add_dataframe):
     es["new_df"] = df
     assert add_dataframe.called
     add_dataframe.assert_called_with(dataframe=df, dataframe_name="new_df")
+
+
+def test_latlong_nan_normalization(latlong_df):
+    latlong_df.ww.init(
+        name="latLong", index="idx", logical_types={"latLong": "LatLong"}
+    )
+    latlong_df.sort_values(by="idx", inplace=True)
+
+    dataframes = {"latLong": (latlong_df,)}
+
+    relationships = []
+
+    es = EntitySet("latlong-test", dataframes, relationships)
+
+    normalized_df = to_pandas(es["latLong"])
+
+    expected_df = to_pandas(
```

Comment:
to_pandas given a pandas dataframe immediately returns the dataframe so we can skip it

Reply:
@rwedge I am parameterizing pandas, dask, and scala. Is there a way I can check which type it is and only run to_pandas if its not pandas?

Reply:
arg! yes of course, thanks